src/OnboardingSPA/pages/Steps/DesignThemeStyles/Preview/stylesheet.scss

```diff
@@ -0,0 +1,97 @@
+.theme-styles-preview {
+    flex: 1;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    background-color: #EEEEEE;
+
+    &__checkbox {
+        margin: 20px 0px;
+        width: 70%;
+        display: inline;
+
+        .components-checkbox-control {
+            &__checked {
+                height: 32px;
+                width: 32px;
+                left: 0px;
+                top: 0px;
+            }
+
+            &__input-container {
+                height: 32px;
+                width: 32px;
+            }
+
+            &__input[type=checkbox] {
+                height: 32px;
+                width: 32px;
+
+                &:focus {
+                    box-shadow: none;
+                }
+            }
+        }
+
+        .components-base-control {
+            &__field {
+                display: flex;
+                flex-direction: row;
+            }
+        }
+
+        &__label {
+
+            margin-left: 15px;
+
+            &__hint {
+                color: #757575;
+                display: block;
+                font-style: italic;
+                margin-top: 5px;
+            }
+        }
+
+    }
+
+    &__title-bar {
+        width: 70%;
+        height: 15px;
+        display: flex;
+        align-items: center;
+        background-color: #cccccc;
+        justify-content: space-between;
+        border: 1px solid transparent;
+
+        &__browser {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+
+            &__dot {
+                background-color: #989EA7;
+                width: 8px;
+                margin: 3px;
+                height: 8px;
+                border-radius: 50%;
+                display: inline-block;
+            }
+        }
+    }
+
+    &__live-preview-container {
+        position: relative;
+        width: 70%;
+        align-items: center;
+        border: 1px solid #e3dfdf;
```

Comment:
Can you please add a `margin-bottom` also? The preview seems to be touching the bottom of the page and there is no clear end.